README.md

```diff
@@ -60,18 +60,48 @@ where lines need to be commented in and the paths modified if cmake cannot
 
 
 ***
+
```

Comment:
@willGraham01 
Would you mind just re-reviewing this file please? Check I'm making sense...